data/net_charge/net_charge.yml

```diff
@@ -0,0 +1,6 @@
+name: net_charge
+language: python
+prompt: net_charge.txt
+label: net_charge.py
+context: python
```

Comment:
This might need a different context.

Reply:
I think python is fine. It is possible to use mdtraj or rdkit here but this can also be done with simple python since sequence and residue info is just strings. Thoughts @navneeth3005 ?